faqs/stocks/which-screener.md

```diff
@@ -9,75 +9,122 @@ description: >-
 
 ## Introduction
 
-Even though screeners are useful in finding out the investment opportunities which suits your criteria and to monitor the existing investment, it might not be a good idea to take a screener at its face value.
+Though stock-screeners calculate a lot of things, you must do you own calculations too. You will miss a lot of good opportunities or make some expensive mistakes if you consider only the pre-calculated values.
 
-You might see an opportunity or a red-flag from these screeners. However, it could easily just be a calculation error, or usage of incorrect formula, or not removing an outlier.
+The objective of this article is not to demean the websites but to show the importance of your own calculations.
 
-This article is to sensitize you that you might have to do more research rather than just going with the presented data from the UI of a screener.
+----
```

Comment:
Maybe add a line here, about how we'd now consider dividend yield as an example.

Reply:
I have rephrased the whole paragraph now.

I have tried to keep the paragraphs concise to improve their chances of being read. Thus I think we can avoid repeating about dividend yield here.